PluginCore/PluginCore/Helpers/FileHelper.cs

```diff
@@ -81,7 +81,8 @@ public static void WriteFile(String file, String text, Encoding encoding, Boolea
         {
             Boolean useSkipBomWriter = (encoding == Encoding.UTF8 && !saveBOM);
             if (encoding == Encoding.UTF7) encoding = new UTF7EncodingFixed();
-            using (StreamWriter sw = useSkipBomWriter ? new StreamWriter(file, false) : new StreamWriter(file, false, encoding))
+            using (FileStream fs = new FileStream(file, FileMode.OpenOrCreate))
```

Comment:
Our problem right now is that the file is not truncated if it already exists. `FileMode.OpenOrCreate` works like `Open` if it exists, and `CreateNew` if not, which is not what we want. `FileMode.Create` (which is what `StreamWriter` uses internally) does `Truncate` if exists, and `CreateNew` if not. However, `Create` seems to not work for hidden files.